src/core/components.py

```diff
@@ -106,6 +107,22 @@ def get_base_component_dict(self):
         }
 
 
+class Metrics(Component):
+    __slots__ = [
+        "metrics",
+    ]
+
+    def __init__(self, metrics='Metrics', **kwargs):
```

Comment:
Only suggestion is putting a docstring specifying what metrics param is expected to be

Reply:
That or change the slot to `label` so it's clear